.pre-commit-config.yaml

```diff
@@ -13,6 +13,7 @@ repos:
     rev: 21.7b0
     hooks:
     -   id: black
+        language_version: python3
```

Comment:
not needed: it uses the `default_language_version` (supposed to be 3.7)

Reply:
When I revert to the python3.7 for both of your suggestions, I get a commit error:

"git" commit -F "C:\Users\Benjamin\Documents\Git\geoapps\.git\COMMITMESSAGE"
[INFO] Installing environment for https://github.com/psf/black.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('c:\\users\\benjamin\\anaconda3\\envs\\geoapps\\python.exe', '-mvirtualenv', 'C:\\Users\\Benjamin\\.cache\\pre-commit\\repotomysriv\\py_env-python3.7', '-p', 'python3.7')
return code: 1
expected return code: 0
stdout:
    RuntimeError: failed to find interpreter for Builtin discover of python_spec='python3.7'

stderr: (none)
Check the log at C:\Users\Benjamin\.cache\pre-commit\pre-commit.log
Done

Press Enter or Esc to exit...

Reply:
https://github.com/pre-commit/pre-commit/issues/1375
^^^ this is what led us to try python3 instead of python3.7

Reply:
maybe it requires some python 3.7 in the path?

well, I guess it is OK with Python 3 then. We will see...
I think the tool introducing syntax change is pyupgrade, and we tell this one to be compatible with 3.7

so keep global `default_language_version` as python3, but remove `language_version` here

Reply:
Was able to commit with your recommended changes, but doesn't seem like it's triggering the tests still.